core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfig.java

```diff
@@ -27,8 +26,6 @@
     private Map<String, OptimizelyExperiment> experimentsMap;
     private Map<String, OptimizelyFeature> featuresMap;
     private String revision;
-
-    @JsonIgnore
```

Comment:
I meant to remove @JsonIgnore redundant below at "getDatafile()".
Is it ok to remove @JsonIgnore for datafile completely?